scripts/1.4.0/README.md

```diff
@@ -1,25 +1,14 @@
-# Scripts for v1.3.12
-
-### Upgrade (only for Canary)
-
-Required to upgrade the Canary from any version above 1.3.4 to 1.3.12.
-
-This script is needed to be able to discover DBMSs installed with older versions of Desktop correctly. Without running this, you'll see the DBMSs in your projects, but won't be able to start them. The script is only changing Desktop configuration and is not modifying any data.
-
-- Start the upgraded version of Neo4j Desktop.
-- Once it's done loading close the application.
-- Run the script.
-- Start Neo4j Desktop again.
+# Scripts for v1.3.12 (released as v1.4.0)
```

Comment:
Do we still need to include version 1.3.12 here considering we changed the name of the folder? And considering there's no 1.3.12 in production.

Reply:
yeah this can be changed 👍